### DISCUSSION TITLE
Workspaces Browser Extension

### DISCUSSION THREAD

**Discussion by JVariance** [Godiesc/firefox-one#19] [Show and tell]
<img src="https://github.com/Godiesc/firefox-one/assets/62245444/adca4d49-ad0b-4e7e-ad62-046340d15eb0" width="96"/>

I am working on a browser extension for workspaces similar to Opera One's (without utilizing Firefox' Multi-Account-Containers). (https://github.com/JVariance/Firefox-Workspaces).

In the image you see that the headline "Workspaces v" overflows. Can you fix this?

**Comment by Godiesc** [comment]
hi,
to delete the header you can add this css:
```
#sidebar-box #sidebar-header {
    visibility: collapse;
    display: none;
}
```

With the extension id or name you can delete only on that extension, for example:
```
#sidebar-box[sidebarcommand*="tabcenter"] #sidebar-header {
    visibility: collapse;
    display: none;
}
```
